seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/config/JdbcOptions.java

```diff
@@ -152,18 +152,39 @@ public interface JdbcOptions {
                     .defaultValue(true)
                     .withDescription(
                             "is the primary key updated when performing an update operation");
+
+    Option<JdbcSinkConfig.WriteMode> WRITE_MODE =
+            Options.key("write_mode")
+                    .enumType(JdbcSinkConfig.WriteMode.class)
+                    .defaultValue(JdbcSinkConfig.WriteMode.SQL)
+                    .withDescription("write mode: SQL/COPY/COPY_SQL/MERGE/COPY_MERGE");
+
+    String REPLACE_TARGET_TABLE_NAME_KEY = "${target_table}";
```

Comment:
Why not reuse https://github.com/apache/seatunnel/blob/757641bada16cf59400723d0720124abee3d1e26/seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/TablePlaceholder.java#L32?
And put update the doc